frontend/src/stores/feature-flag.ts

```diff
@@ -204,9 +211,11 @@ export const useFeatureFlagStore = defineStore(FEATURE_FLAG, {
      * are read in the corresponding `initFromCookies` method.
      */
     writeToCookie() {
+      const secure = useRuntimeConfig().public.deploymentEnv === PRODUCTION
```

Comment:
What's the logic here? Shouldn't it also be applied to staging? Why not, if not?

Reply:
I think the Secure cookies didn't work well locally, so the check was for production build (production/staging).

Checked again, and the cookies work fine on localhost, so I used `secure` for all envs.

Reply:
Setting cookies to `secure` broke a unit test, but I mocked the value to set it to `secure: false`.